io/plugins/eu.esdihumboldt.hale.io.appschema/src/eu/esdihumboldt/hale/io/appschema/writer/internal/AbstractPropertyTransformationHandler.java

```diff
@@ -255,12 +257,14 @@ private ChainConfiguration findLongestNestedPath(List<ChildContext> targetProper
 	 * @param featureType the target feature type
 	 * @param mappingName the target feature type's mapping name (may be
 	 *            <code>null</code>)
+	 * @param context
 	 */
-	protected void handleAsFeatureGmlId(TypeDefinition featureType, String mappingName) {
+	protected void handleAsFeatureGmlId(TypeDefinition featureType, String mappingName,
+			AppSchemaMappingContext context) {
```

Comment:
The JavaDoc for the new `context` parameter is missing (4x in the file).

Reply:
Fixed, thanks.